Trees Excercises.py

```diff
@@ -0,0 +1,77 @@
+#Trees Excercises
+
+#Tree Construction
+class TreeNode: 
+    def __init__(self, left, right, value):
+        self.left = left
+        self.right = right
+        self.value = value
+
+class Tree:
+    def __init__ (self, root):
+        self.root = root
+
+    def add_node (self, value):
+        node = TreeNode(
+            left = None,
+            right= None,
+            value = value
+        )
```

Comment:
Good idea to create a TreeNode instance within the add_node function, it will save the callers the trouble of doing it on their end